x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/utils.ts

```diff
@@ -80,6 +82,14 @@ export function crawlRequestServerToClient(crawlRequest: CrawlRequestFromServer)
   };
 }
 
+export function crawlConfigServerToClient(crawlConfig: CrawlConfigFromServer): CrawlConfig {
```

Comment:
In the future, it'd be great to have a common adapter on API calls that would convert response keys to the correct casing automatically.